src/packages/apollo/src/plugins/cors.ts

```diff
@@ -1,7 +1,22 @@
 import { ApolloServerPlugin } from '@apollo/server';
 import { logger } from '@exogee/logger';
 
-export const Cors: ApolloServerPlugin = {
+// @todo does anyone need this legacy code?
+const defaultValidateOrigin = (origin: string) => {
+	const rootDomain = process.env.ROOT_DOMAIN;
```

Comment:
I dont think we use this, shall we remove it?

Reply:
im not sure what `easy-` is

Reply:
happy to remove it, it's just the logic in the existing code

Reply:
yeah i think lets tidy it up we can always add back if its needed